src/CONST.ts

```diff
@@ -523,6 +523,16 @@ const CONST = {
             shortcutKey: 'Tab',
             modifiers: [],
         },
+        DEBUG: {
+            descriptionKey: 'openDebug',
```

Comment:
@dominictb can we please add the openDebug key to the translations files ? , so that user can see the description for the CMD+D shortcut 

https://github.com/Expensify/App/blob/d0b355282ea9ddfd3780696b973fe0eb9448d72a/src/languages/en.ts#L2786-L2791

Reply:
And also please add the attachments to checklist for native devices that reflect the following https://github.com/Expensify/App/issues/43256#issuecomment-2154610196